### PR TITLE
Add feature for SVG attributes conversion

### PR DESCRIPTION
**Pull request by tsuyoshiwada**
Hi, Thank you for useful library :smiley:

I wanted to convert the attributes of SVG like #135.
However, when I actually convert it I got the following output.

```diff
- Actual: fill-rule, stroke-width
+ Expected: fillRule, strokeWidth
```

So we added `SVGDOMPropertyConfig` as an attribute mapping target as well as` HTMLDOMPropertyConfig`.

## Related issues

* [Convert hyphenated attributes into camelCase #135](https://github.com/reactjs/react-magic/issues/135)

### CODE REVIEW

**Comment by ritz078** [comment]
Any chance of this getting merged soon ?

**Comment by tsuyoshiwada** [comment]
Hi @ritz078 .

If you are in a hurry, htmltojsx that corresponds to SVG is released in Scoped Package, so please try using it :smiley:

https://github.com/tsuyoshiwada/htmltojsx

**Comment by Daniel15** [comment]
Sorry, I completely missed this... I'll take a look soon!

**Comment by tsuyoshiwada** [comment]
Hi @Daniel15, Thanks !

**Comment by ritz078** [comment]
Thanks @tsuyoshiwada . FYI I am using it on https://transform.now.sh/html-to-jsx

File: src/htmltojsx.js [hunk 1]
```diff
@@ -562,12 +579,9 @@ StyleParser.prototype = {
    */
   toJSXString: function() {
     var output = [];
-    for (var key in this.styles) {
-      if (!this.styles.hasOwnProperty(key)) {
-        continue;
-      }
-      output.push(this.toJSXKey(key) + ': ' + this.toJSXValue(this.styles[key]));
-    }
+    eachObj(this.styles, function(key, value) {
```
Comment (Daniel15) [inline src/htmltojsx.js]: What's the purpose of this change?

Comment (tsuyoshiwada) [inline src/htmltojsx.js]: We have refactored the iteration of the object using the newly created `eachObj` helper.
I'm sorry I made a confusing change... :cry:

Comment (Daniel15) [inline src/htmltojsx.js]: It's totally fine, don't worry @tsuyoshiwada 😄  I was just wondering about why we couldn't just continue to use a regular `for` loop here.

Comment (tsuyoshiwada) [inline src/htmltojsx.js]: Thank you! It was worried whether it became a meddle, but I am relieved :smiley:

**Comment by Daniel15** [comment]
Released as 0.3.0 😃

**Comment by tsuyoshiwada** [comment]
Hi @Daniel15, Thank you :smile: I checked that I was working.
